config/database.yml

```diff
@@ -19,8 +19,6 @@ development:
   encoding: unicode
   database: receta_development
   pool: 5
-  username: receta
```

Comment:
Actually, it is best not even to have database.yml in the repository, in most cases.
I just have my database set up for dev where it is not needed, however might be best to 
leave username in for most people.